### PR TITLE
[MRG] DMPSequence: Trajectory Generator

### PR DESCRIPTION
**Pull request by ChristophPetzoldt**
Generate the trajectory represented by the DMP-sequence in open loop.

### CODE REVIEW

File: bolero/representation/dmp_sequence.py [hunk 1]
```diff
@@ -323,3 +323,65 @@ def can_step(self):
...
+        last_t = 0.0
+        last_y = np.copy(self.subgoals[0])
+        last_yd = np.copy(self.subgoal_velocities[0])
+        last_ydd = np.zeros(self.n_task_dims)
+
+        y = np.empty(self.n_task_dims)
+        yd = np.empty(self.n_task_dims)
+        ydd = np.empty(self.n_task_dims)
+
+        Y = []
```
Comment (AlexanderFabisch) [inline bolero/representation/dmp_sequence.py]: I think we can pre-allocate the complete numpy arrays here already.

File: bolero/representation/dmp_sequence.py [hunk 2]
```diff
@@ -323,3 +323,65 @@ def can_step(self):
         """Returns true if step() can be called again, false otherwise."""
         return len(np.where(self.steps <= self.split_steps)[0]) > 0
 
+    def trajectory(self):
+        """Generate trajectory represented by the DMP in open loop.
```
Comment (AlexanderFabisch) [inline bolero/representation/dmp_sequence.py]: DMP -> sequence of DMPs

File: bolero/representation/dmp_sequence.py [hunk 3]
```diff
@@ -323,3 +323,65 @@ def can_step(self):
         """Returns true if step() can be called again, false otherwise."""
         return len(np.where(self.steps <= self.split_steps)[0]) > 0
 
+    def trajectory(self):
```
Comment (AlexanderFabisch) [inline bolero/representation/dmp_sequence.py]: We need unit tests for this function [here](https://github.com/rock-learning/bolero/blob/master/bolero/representation/test/test_dmp_seq.py).

Comment (ChristophPetzoldt) [reply bolero/representation/dmp_sequence.py]: Added simple subgoal test

File: bolero/representation/dmp_sequence.py (hunk 1 above)
Comment (ChristophPetzoldt) [reply bolero/representation/dmp_sequence.py]: Pre-allocated the trajectory output arrays :+1:

**Pull request merged**